numpy/testing/nosetester.py

```diff
@@ -413,27 +402,55 @@ def test(self, label='fast', verbose=1, extra_argv=None,
             for warningtype in raise_warnings:
                 warnings.filterwarnings('error', category=warningtype)
             # Filter out annoying import messages.
-            warnings.filterwarnings('ignore', message='Not importing directory')
-            warnings.filterwarnings("ignore", message="numpy.dtype size changed")
-            warnings.filterwarnings("ignore", message="numpy.ufunc size changed")
-            warnings.filterwarnings("ignore", category=np.ModuleDeprecationWarning)
-            warnings.filterwarnings("ignore", category=FutureWarning)
```

Comment:
Hah... actually, scipy seems to never runs in development mode, so that the warnings are never set to raise, but only printed out. As such, you might get a print or so more, but actually don't even see any FutureWarning, which is almost surprising.